cmd/auth_test.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAuthRootCommand(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml := testUtil.NewTestConfig()
+	err := afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	expectedOut := `Handles authentication to the Astronomer Platform
+
+Usage:
+  astro auth [command]
+
+Available Commands:
+  login       Login to Astronomer services
+  logout      Logout of Astronomer services
+
+Flags:
+  -h, --help   help for auth
+
+Use "astro auth [command] --help" for more information about a command.
+`
+	_, output, err := executeCommandC(nil, "auth")
```

Comment:
@ashb @schnie 
For instance i can check only:
`assert.NoError(err)`, and skip comparing with full output of command or check something like:
`assert.NoError(err)`
`assert.Contains(output, "astro auth")` if somebody forgot to register command.

Reply:
> if somebody forgot to register command
That feels to me like it is something that is more easily caught at the "unit" test level/by inspecting the created command objects, or just by calling the sub commands we are interested in.

I can't think of many circumstances where a test like this one as it is currently would catch a bug, but we would have to "keep it in sync" with help text and sub commands.

I think `assert.NoError(err)` would be good enough for cases like this.

(What is the `_` return value we are ignoring btw? Isn't this also what `executeCommand()` func does?)

Reply:
>(What is the _ return value we are ignoring btw? Isn't this also what executeCommand() func does?)
correct, i'll fix asap.

Reply:
fixed in 0c08289